### ISSUE TITLE
Audio out support

### ISSUE THREAD

**Issue by eliawk** [schollz/autojack#1]
Hi @schollz,
thanks for this amazing mod!
it would be possibile to edit the script to have also audio out through usb?

**Comment by schollz** [comment]
Yeah!! It should be possible, probably add something like this: https://github.com/schollz/autojack/blob/main/lib/mod.lua#L31

Add those lines for alsa out. 

But the device should be able to handle USB audio. What will you plug the norns into?

**Comment by eliawk** [comment]
@schollz tx for the anwwer., this evening I will give ti a try.
I would like to connect the norns to the Elektron Analog Heat, but atm also the audio in seems to not working.
maybe because I've a Fates instead fo a regular Norns?

**Comment by schollz** [comment]
hm I don't think so but maybe. Its really hard to trouble shoot USB audio. The best way is to find the relevant commands in the mod and run them yourself in a terminal and see messages you get if any and if it works or not

**Comment by eliawk** [comment]
@schollz it was my fault, now your mod as it is now it's works!! so now I can get the audio through usb.

**Comment by synthi** [comment]
> Yeah!! It should be possible, probably add something like this: https://github.com/schollz/autojack/blob/main/lib/mod.lua#L31
> 
> Add those lines for alsa out.
> 

Can’t understand what to do there… maybe it could be a new branch with USB audio out?
Thanks!
